util/math128.h

```diff
@@ -236,6 +246,18 @@ inline Unsigned128 DownwardInvolution(Unsigned128 v) {
          DownwardInvolution(Upper64of128(v) ^ Lower64of128(v));
 }
 
+template <typename A>
```

Comment:
Just curious which one of these two templates will be more specialized for a BitwiseAnd(Unsigned128, Unsigned128) call.

Reply:
Nice catch. There should be no need for using `BitwiseAnd` vs. just `&` in that case, unless you're doing some template work. If someone hits that, they can fix it. The only reason I added 128-bit support here was to fit into the existing unit test infrastructure.